shared/js/gzip/gzip.js

```diff
@@ -0,0 +1,39 @@
+/* exported Gzip */
+'use strict';
+
+var Gzip = {
+
+  WORKER_URL: 'shared/js/gzip/gzip_worker.js',
```

Comment:
I see that in the test you just add a / at the start of this URL.  Does it work here if you just hardcode the /?  It would be easiest if you didn't need this property at all.  But if you do need it, please add a comment explaining that it is stored in a property so that tests can modify it.